application/src/main/java/org/togetherjava/tjbot/features/filesharing/FileSharingMessageListener.java

```diff
@@ -120,29 +133,21 @@ private boolean isAttachmentRelevant(Message.Attachment attachment) {
     }
 
     private void processAttachments(MessageReceivedEvent event,
-            List<Message.Attachment> attachments) {
-
-        Map<String, GistFile> nameToFile = new ConcurrentHashMap<>();
+            List<Message.Attachment> attachments) throws IOException {
+        GHGistBuilder gistBuilder = new GitHubBuilder().withOAuthToken(gistApiKey)
+            .build()
+            .createGist()
+            .public_(true)
```

Comment:
Sure public is correct? The old code had a `false` in the constructor. Please double check. Ideally, if possible, I would want unlisted.

Reply:
others can see non-public(private) gists?

Reply:
id like to have unlisted, if thats an option. not sure about private, can u quickly check what the old code did?

Reply:
dk about unlisted, but setting public to false is working fine

Reply:
done